### PR TITLE
Send All (1e-8) from Blind to Public gives error fix

### LINKED ISSUE #776: Send All (1e-8) from Blind to Public gives error
Please find the attached screenshot 
screen shot 2018-01-31 at 4 07 18 pm
screen shot 2018-01-31 at 4 07 31 pm
screen shot 2018-01-31 at 4 07 48 pm

### PR DESCRIPTION
**Pull request by vikas-cis**
#776

### CODE REVIEW

File: src/app/wallet/wallet/send/send.component.ts [hunk 1]
```diff
@@ -120,6 +121,11 @@ export class SendComponent {
       this.send.validAmount = false;
       return;
     }
+
+    if (this.send.amount === 1e-8) {
```
Comment (anandsinghparihar) [inline src/app/wallet/wallet/send/send.component.ts]: @vikas-cis  can we set amount `0` if the amount equals to `1e-8` ?

Comment (vikas-cis) [reply src/app/wallet/wallet/send/send.component.ts]: @anandsinghparihar its not `0` it is 1 SAT I guess which is a valid amount. So I think we don't need to show `0`

File: src/app/wallet/wallet/send/send.component.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ export class SendComponent {
     this.send.input = 'balance';
     if (this.type === 'balanceTransfer') {
       this.send.toAddress = '';
+      this.send.output = 'blind_balance'
```
Comment (kewde) [inline src/app/wallet/wallet/send/send.component.ts]: why this?

Comment (vikas-cis) [reply src/app/wallet/wallet/send/send.component.ts]: @kewde yeah if you are on "send payment" and check the `blind` and then moved to "Balance Transfer" in that case `public` is checked on both side which is incorrect.

**Pull request merged**